Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>bin/\r\nobj/\r\n/packages/\r\nriderModule.iml\r\n/_ReSharper.Caches/\r\n.idea/\r\n.idea.TripsProject/
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 4451a070ab5c2d23a023851fb616de96dc29a4de)
+++ b/.gitignore	(date 1764155580204)
@@ -4,4 +4,6 @@
 riderModule.iml
 /_ReSharper.Caches/
 .idea/
+.idea.TripsProject/
+.idea/
 .idea.TripsProject/
\ No newline at end of file
